_posts/en/posts/2016-01-25-segwit-benefits.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+name: segwit-benefits
+id: en-segwit-benefits
+title: Segregated Witness Benefits
+permalink: /en/2016/01/25/segwit-benefits
+version: 1
+excerpt: >
+  This page summarises some of the benefits of segregated witness that
+  go beyond simply increasing the capacity of the block chain.
+---
+{% include _toc.html %}
+
+The Segregated Witness soft-fork (segwit) includes a wide range of
+features, many of which are highly technical. This page summarises
+some of the benefits of those features that go beyond simply increasing
+the capacity of the block chain.
+
+## Malleability Fixes
+
+Bitcoin transactions are identified by a 64-digit hexadecimal hash
+called a transaction identifier (txid)
+which is based on both the coins being spent and on who
+will be able to spend the results of the transaction.  Unfortunately,
+the way the hash is calculated allows anyone to make small
+modifications to the transaction that will not change its meaning,
+but will change the hash.  This is called third-party malleability.
+Another type of malleability is possible if one of the signers of the
+transaction revises their signature; again the the transaction remains
+valid and pays the same amounts to the same addresses, but as the
+transaction hash incorporates the signature, the hash changes. This is
+signer malleability.
+
+For example, you could submit a transaction with hash
+ef74...c309 to the
+network, but instead find that a node or miner on the network modifies the transaction slightly, and that the transaction is confirmed under
+the hash 683f...8bfa
+instead.
+
+Segwit prevents third-party and signer malleability by allowing Bitcoin users to move the
```

Comment:
How about "scriptSig malleability" -- would that be accurate ? https://en.bitcoin.it/wiki/Transaction_Malleability